string git_SHA
string script_path
string exp_description
string script_code
